<launch>
    <arg name="object" default="line" />
    <node name="color_calibrate" type="color_calibrate" pkg="color_calibration" respawn="true" />
    <node name="fusion_node" type="fusion_node" pkg="vision_fusion" respawn="true" />
</launch>